#! /bin/sh /usr/share/dpatch/dpatch-run
## 04-makefile.dpatch by Simo Kauppi <swk@nic.fi>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: This patch adds the LDFLAGS to linker in Makefile.common.in to pass
## DP: the -WL,-z defs parameters.
## DP: This patch also removes the libart from the lib/Makefile.in and
## DP: src/Makefile.in, because we use the Debian libart instead of the
## DP: one included in the upstream tar ball.

@DPATCH@
diff -urNad swftools-0.9.0~/Makefile.common.in swftools-0.9.0/Makefile.common.in
--- swftools-0.9.0~/Makefile.common.in	2009-04-12 23:27:16.000000000 +0000
+++ swftools-0.9.0/Makefile.common.in	2009-05-06 03:38:30.000000000 +0000
@@ -11,8 +11,8 @@
 DEP=@CC@ -MM $(DEFS) $(INCLUDES) @CPPFLAGS@
 C=@CC@ -c $(DEFS) $(INCLUDES) @CPPFLAGS@ @CFLAGS@
 CC=@CXX@ -c $(DEFS) $(INCLUDES) @CPPFLAGS@ @CXXFLAGS@
-L=@CC@ $(DEFS)
-LL=@CXX@ $(DEFS)
+L=@CC@ $(DEFS) $(LDFLAGS)
+LL=@CXX@ $(DEFS) $(LDFLAGS)
 AR=@AR@
 LIBTOOL=@LIBTOOL@
 SC=$(LIBTOOL) @CC@ -c $(DEFS) $(INCLUDES) @CPPFLAGS@ @CFLAGS@
